src/api/validation/problem.validation.ts

```diff
@@ -32,10 +32,16 @@ const problemValidation = (data: any): ValidationResult => {
     hidden: boolean().required(),
     language: string().required(),
     dueDate: date().iso().required(),
+    author: string().allow(''),
+    difficulty: string().allow(''),
+    dateCreated: date().iso(),
+    lastModified: date().iso(),
+    footer: string().allow(''),
```

Comment:
what is this footer for?

Reply:
It's the code footer. I think the frontend was modified to sometime use it so it won't run without this in.